dcos/cosmospackage.py

```diff
@@ -562,8 +583,11 @@ def _get_header(request_type, version):
     :rtype: str
     """
 
-    return ("application/vnd.dcos.package.{}+json;"
-            "charset=utf-8;version={}").format(request_type, version)
+    if request_type == 'add-response':
+        return 'application/vnd.dcos.universe.package+json;charset=utf-8;version=v3'
```

Comment:
Why are we mapping `add-reponse` to `.../vnd.dcos.universe.package...`? This should not be a valid type returned by Cosmos. The add command in Cosmos should  always return a `application/vnd.dcos.package.add-response+json;charset=utf-8;version=v1`. cc @cruhland